content/blog/tidymodels-2021-q1/index.Rmd

```diff
@@ -0,0 +1,141 @@
+---
+output: hugodown::hugo_document
+
+slug: tidymodels-2021-q1
+title: Catch up with tidymodels
+date: 2021-03-10
+author: Julia Silge
+description: >
+    Recent tidymodels releases offer new functions for easier model building and
+    resampling, along with a new package for resampling spatial data.
+
+photo:
+  url: https://unsplash.com/photos/4Zk45jNyQS4
+  author: Timo Wielink
+
+# one of: "deep-dive", "learn", "package", "programming", or "other"
+categories: [package] 
+tags: [tidymodels, parsnip, rsample]
+---
+
+<!--
+TODO:
+* [ ] Pick category and tags (see existing with `hugodown::tidy_show_meta()`)
+* [ ] Find photo & update yaml metadata
+* [ ] Create `thumbnail-sq.jpg`; height and width should be equal
+* [ ] Create `thumbnail-wd.jpg`; width should be >5x height
+* [ ] `hugodown::use_tidy_thumbnails()`
+* [ ] Add intro sentence
+* [ ] `usethis::use_tidy_thanks()`
+-->
+
+```{r include=FALSE}
+library(ggplot2)
+theme_set(theme_minimal())
+```
+
+
+There have quite a number of updates and new developments in the [tidymodels](https://www.tidymodels.org/) ecosystem since our [last blog post in December](https://www.tidyverse.org/blog/2020/12/finetune-0-0-1/)! Since that post, tidymodels maintainers have published seven CRAN releases of existing packages:
```

Comment:
I think it'd be a good idea here to include a one or two sentence description of tidymodels, just in case folks reading the post don't already know.